manifests/site.pp

```diff
@@ -11,20 +11,8 @@
 
 ## Active Configurations ##
 
-# PRIMARY FILEBUCKET
-# This configures puppet agent and puppet inspect to back up file contents when
-# they run. The Puppet Enterprise console needs this to display file contents
-# and differences.
-
-# Define filebucket 'main':
-filebucket { 'main':
-  #server should point to one master that will be the file bucket
-  server => "${settings::server}",
-  path   => false,
-}
-
-# Make filebucket 'main' the default backup location for all File resources:
-File { backup => 'main' }
+# Disable filebucket by default for all File resources:
+File { backup => false }
```

Comment:
Are we recommending all clients give up on filebucket? I would agree with that, but how are we messaging that? Will this end up with people expecting and foolishly depending on the backup?